app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -6,7 +6,10 @@ class Vmware < ComputeResource
     include ComputeResourceConsoleCommon
     include ComputeResourceCaching
 
-    validates :user, :password, :server, :datacenter, :presence => true
+    validates :user, :password, :datacenter, :presence => true
+    validates :server, :format => { :with => URI.regexp }, :presence => true,
+              :url_schema => ['http', 'https']
```

Comment:
@swapab : Yes, please. Basically we want to support simple hostnames. "vcenter.example.com". Extra points for adding a test for that.

```
should allow_values('vcenter.example.com', 'vcenter').for(:server)
```